src/frontend/src/lib/components/hero/HeroContent.svelte

```diff
@@ -33,7 +33,7 @@
 </script>
 
 <div
-	class="class:from-blue-ribbon-light flex h-full w-full flex-col content-center items-center justify-center rounded-[40px] bg-blue-ribbon bg-gradient-to-b from-blue-ribbon via-absolute-blue bg-size-200 bg-pos-0 p-6 text-center text-white transition-all duration-500 ease-in-out"
```

Comment:
the removed one is unrelated: it has no effect to the class and was there by mistake